### ISSUE TITLE
[ERROR] `net.minecraft.entity.passive.EntityBat cannot be cast to net.minecraft.entity.player.EntityPlayer`

### ISSUE THREAD

**Issue by CodeByAidan** [PrinceOfAmber/Cyclic#2292]
Minecraft Version: 1.12.2

Forge Version: forge-1.12.2-14.23.5.2854

Mod Version: 1.20.2 (Cyclic-1.12.2-1.20.2.jar)

Single Player or Server: Server

Describe problem (what you were doing; what happened; what should have happened):
Honestly, I have 0 idea what happened. My server was working fine, until I guess a bat spawned, and my server crashed. That's really it. I've tried removing the mod, and installing it again, to no avail. My version was part of the modpack "Stoneblock".

Log file link:
https://mcpaste.io/b6ac256d96f74079

**Comment by Lothrazar** [comment]
the most recent version of the mod under the minecraft "1.12.2" era is 'Cyclic-1.12.2-1.20.14" , so i would first try updating that, from .2 to .14 and see if it still happens

**Comment by CodeByAidan** [comment]
I deleted the server 

![crying-emoji-dies](https://github.com/Lothrazar/Cyclic/assets/67598470/7d8edf4a-f0f0-4dd6-af05-8838cf911ea8)

**Comment by Xyloxy** [comment]
Hello,

I've encountered the same issue on SkyFactory 4 (version 4.2.4). Updating cyclic from 1.20.3 to 1.20.14 fixed the issue, and caused no additional bugs to be present on the server during the quick testing that I've done.

For additional context the error was encountered when one of the players threw a saturation potion at other player.

Therefore this bug has been fixed already, thanks for updating this mod for older versions!
